k8s/sonarqube/Makefile

```diff
@@ -0,0 +1,109 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+
+TAG ?= 7.6
+METRICS_EXPORTER_TAG ?= v0.5.1
+SRCTAG ?= 9.6-kubernetes
+IMAGE_SONARQUBE ?= marketplace.gcr.io/google/sonarqube7:latest
```

Comment:
Still, you want to ask for specific tag here, like 7.6.

Reply:
the same as in all application.